x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/assignment/TrainedModelAssignmentClusterService.java

```diff
@@ -480,57 +525,95 @@ ClusterState createModelAssignment(ClusterState currentState, StartTrainedModelD
         return update(currentState, rebalanceAssignments(currentState, Optional.of(params)));
     }
 
-    private void rebalanceAssignments(
+    void rebalanceAssignments(
```

Comment:
Can this be private? I might have missed it but I don't think we reference it from the tests.

Reply:
seems like yes